stan/math/prim/scal/meta/require_generics.hpp

```diff
@@ -174,11 +177,37 @@ template <typename T, typename... Types>
 using require_any_not_convertible_t = require_all_not_t<
     std::is_convertible<std::decay_t<T>, std::decay_t<Types>>...>;
 
+// Checks if type is something we would use for string_convertible
+template <typename T>
+using require_string_convertible_t
+    = require_t<is_string_convertible<std::decay_t<T>>>;
```

Comment:
Decaying the type before passing it to ```is_string_convertible``` could cause a false negative: https://godbolt.org/z/_8H6rK

Reply:
Oh my! Thanks for the catch. I use decay a lot in the requires library. I may need to look over if this effects other things

Reply:
We both read too quickly haha, gotta use the `_t` on that `std::decay`!

https://godbolt.org/z/ElJ1YM